packages/mdc-checkbox/README.md

```diff
@@ -52,6 +57,7 @@ easily position checkboxes and their labels.
         <path class="mdc-checkbox__checkmark__path" fill="none" stroke="white"
               d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
       </svg>
+    </div>
```

Comment:
I think this part needs to be updated as well with what you did for the first example.